expts/neurips2023_configs/config_large_gcn.yaml

```diff
@@ -15,7 +15,7 @@ accelerator:
           max_num_edges_per_graph: 100
         ipu_dataloader_inference_opts:
           mode: async
-          max_num_nodes_per_graph: 30 # valid max nodes: 51, max_edges: 118
+          max_num_nodes_per_graph: 35 # valid max nodes: 51, max_edges: 118
```

Comment:
Fix for this error:
`AssertionError: Too many nodes. Graph has 905 nodes and max_num_nodes is 900.`